go/vt/vtgate/executor_select_test.go

```diff
@@ -1498,7 +1498,7 @@ func TestStreamSelectIN(t *testing.T) {
 }
 
 func createExecutor(serv *sandboxTopo, cell string, resolver *Resolver) *Executor {
-	return NewExecutor(context.Background(), serv, cell, resolver, false, false, testBufferSize, cache.DefaultConfig, nil, false, querypb.ExecuteOptions_V3)
+	return NewExecutor(context.Background(), serv, cell, resolver, false, false, testBufferSize, cache.DefaultConfig, nil, false, querypb.ExecuteOptions_V3, 0)
```

Comment:
Query serving team may disagree to me, but might be good to set this to a number greater than 0 to help shake out any issues over the next release cycle.

Reply:
This would create nondeterministic replica queries, which can make for flakey tests (we encountered this problem)